cleverhans/future/torch/tests/test_attacks.py

```diff
@@ -27,13 +29,23 @@ def forward(self, x):
     x = torch.matmul(x, self.w2)
     return x
 
+
+class TrivialModel(torch.nn.Module):
+  def __init__(self):
+    super(TrivialModel, self).__init__()
+    self.w1 = torch.tensor([[1.0, -1.0]])
+
+  def forward(self, x):
+    return torch.matmul(x, self.w1)
+
+
 class CommonAttackProperties(CleverHansTest):
 
   def setUp(self):
     super(CommonAttackProperties, self).setUp()
     self.model = SimpleModel()
     self.x = torch.randn(100, 2)
-    self.normalized_x = torch.rand(100, 2) # truncated between [0, 1)
+    self.normalized_x = torch.rand(100, 2)  # truncated between [0, 1)
```

Comment:
This change (and similar ones below it that are seemingly not relevant) were auto generated when I ran autopep8. Nonetheless, these changes should be here anyway